deps/wazuh_testing/wazuh_testing/vulnerability_detector.py

```diff
@@ -3,8 +3,14 @@
 # This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
 
 import re
+import os
+import sqlite3
+import datetime
 from collections import namedtuple
 
+PACKAGES_DB_PATH = "/var/ossec/queue/db/"
+CVE_DB_PATH = "/var/ossec/queue/vulnerabilities/cve.db"
```

Comment:
This creates a dependency on the Wazuh's installation path. Try to detect where is Wazuh installed to initialize this variable with the correct path.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/700/commits/f2c37821e742ba68695a22103e2ab7d3644f50ba